test/run_test.py

```diff
@@ -103,7 +103,6 @@ def skip_test_p(name: str) -> bool:
         'test_kernel_launch_checks',
         'test_metal',
         'test_nnapi',
-        'test_python_dispatch',
         'test_functionalization',
         'test_segment_reductions',
         'test_static_runtime',
```

Comment:
Do all of these tests not run in CI?

Reply:
Yes that's the list of things that don't get run :(

Reply:
cc @bdhirsh you probably want to take functionalization off of this list